challenges/desafio26.js

```diff
@@ -0,0 +1,9 @@
+db.voos.deleteMany(
+  {
+    "empresa.nome": "GOL",
+    $and: [
+    {"passageiros.pagos": {$gte: 5}},
+    {"passageiros.pagos": {$lte: 10}}
+    ]
+  }
+);
```

Comment:
utilizando o $and implicito e tirando chaves duplicadas:
```suggestion
db.voos.deleteMany({ "passageiros.pagos": { $gte: 5, $lte: 10 }, "empresa.nome": "GOL" });
```